frontend/components/PhotoViewer.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import * as PropTypes from "prop-types";
+
+export const SIDES = {
+    CLEANED: "cleaned",
+    FRONT: "front",
+    BACK: "back"
+};
+
+function getValue(dictionary, key, default_val) {
+    let result = dictionary[key];
+    if (typeof result === "undefined") {
+        result = default_val;
+    }
+    return result;
+}
+
+export class PhotoViewer extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    getSource (photoData, displaySide = false) {
+        photoData.propTypes = {
```

Comment:
Thanks for moving this here. We will want to really simplify this function once we start using the new photos — No need for SIDES obj any more, either. Each photo will come with a slide view and a photo view and that's it.